.gitignore

```diff
@@ -1,3 +1,4 @@
+*.nix
```

Comment:
what is this?

Reply:
I had some local scripts that I was keeping in my local repo and trying not to commit. I have them in a folder outside of the repo, so this is no longer necessary.